ci/terraform/oidc/ecc-signing-key.tf

```diff
@@ -96,3 +98,37 @@ data "aws_iam_policy_document" "mfa_reset_signing_key_access_policy" {
     }
   }
 }
+
+
+resource "aws_kms_key" "mfa_reset_jar_signing_key_ecc" {
+  description              = "KMS signing key (ECC) for MFA reset JARs when resetting MFA"
```

Comment:
Maybe something like
"KMS signing key (ECC) for JARs sent from Authentication to IPV for MFA reset"

Reply:
That makes more sense than what I wrote 😄